backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlConstants.java

```diff
@@ -48,6 +48,7 @@ public class GraphqlConstants {
   public static final String MESSAGE = "message";
   public static final String TOKEN = "token";
   public static final String STATUS = "status";
+  public static final String DATA = "data";
 
   public static final String OPERATION = "operation";
   public static final String STAMP = "stamp";
```

Comment:
I do not know if it is possible, but all these look more suitable for an ENUM to me